functions/gcs/sample.txt

```diff
@@ -0,0 +1,14 @@
+Shall I compare thee to a summer's day?
```

Comment:
Shakespeare is an interesting choice; I always go for Alice's Adventures in Wonderland. 😛